components/builder-web/app/app.component.html

```diff
@@ -1,19 +1,14 @@
 <hab-banner></hab-banner>
 <div id="main-nav">
-  <hab-notifications
-    [notifications]="state.notifications.all"
-    [removeNotification]="removeNotification"></hab-notifications>
-  <hab-header
-    [appName]="state.app.name"
-    [isUserNavOpen]="user.isUserNavOpen"
-    [isSignedIn]="user.isSignedIn"
-    [username]="user.username"
-    [avatarUrl]="user.gitHub.get('avatar_url')"
-    [signOut]="signOut"
-    [toggleUserNavMenu]="toggleUserNavMenu"></hab-header>
+  <hab-notifications [notifications]="state.notifications.all" [removeNotification]="removeNotification"></hab-notifications>
```

Comment:
I actually don't love this formatting, but my TSLint autocorrect settings did it automatically. I'm down to change this (although it's probably also a hint that components probably shouldn't take fifty inputs, anyway).